Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -37,10 +38,21 @@ class MongoSwiftTestCase: XCTestCase {
     /// If the environment variable does not exist, this will use a default of "mongodb://127.0.0.1/".
     static var connStr: String {
         if let connStr = ProcessInfo.processInfo.environment["MONGODB_URI"] {
+            if topologyType == .sharded {
+                let uri = mongoc_uri_new(connStr)
```

Comment:
This needs to be freed with `mongoc_uri_destory`.

Reply:
lol jinx (well you beat me to it but I didn't refresh...)

Reply:
man, after all that recent business 🤦‍♂️